cmd/query/app/qualitymetrics/data_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package qualitymetrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+)
+
+func TestGetSampleData(t *testing.T) {
+	data := GetSampleData()
+	assert.NotEmpty(t, data.TraceQualityDocumentationLink, "TraceQualityDocumentationLink should not be empty")
```

Comment:
tip: you don't need these messages in asserts, the assert itself already provides very readable error message with line number